fitIn/Scenario.swift

```diff
@@ -7,15 +7,16 @@
 //
 
 import Foundation
+import AWSDynamoDB
 
 class Scenario {
 
     // Enum for the type of response that the Scenario requires
     // MAY potentially use associated values for storing of answer type
-    enum responseType {
-        case yesOrNo(Int)
-        case slider(Int)
-        case multipleChoice(Int)
+    enum ScenarioType : Int {
```

Comment:
Why make the enum inherit from int?

Reply:
I was hoping to be able to treat the enum like an int before i learned  of .rawValue, its probably useless now